tcms/rpc/tests/test_classification.py

```diff
@@ -45,3 +47,26 @@ def verify_api_with_permission(self):
     def verify_api_without_permission(self):
         with self.assertRaisesRegex(ProtocolError, "403 Forbidden"):
             self.rpc_client.Classification.filter({})
+
+
+class TestClassificationCreate(APIPermissionsTestCase):
+    permission_label = "management.add_classification"
+
+    def _fixture_setup(self):
+        super()._fixture_setup()
+        self.product = ProductFactory()
+
+    def verify_api_with_permission(self):
+        result = self.rpc_client.Classification.create({"name": "API-CATEGORY",
+                                                        "product": self.product.id})
```

Comment:
Classification objects don't have a field `product`, only `name`. Looks like a copy-paste error. 

Interestingly enough the test didn't fail ?!?.


While at it adjust the name to something different, not "CATEGORY".

Reply:
That really was too much copy/paste, yes. 
It looks like any unknown keys are just ignored - confirmed through ipython:
```
In [26]: kiwi.exec.Classification.create({'name': 'extra', 'product': 1, 'drinks': False})
Out[26]: {'id': 5, 'name': 'extra'}

In [27]: kiwi.exec.Classification.filter({'name': 'extra'})
Out[27]: [{'id': 5, 'name': 'extra'}]
```